src/elements/content-sidebar/ContentSidebar.scss

```diff
@@ -43,7 +43,7 @@ $sidebarWidth: $sidebarTabsWidth + $sidebarContentWidth;
             &.bcs-is-open {
                 position: absolute;
                 bottom: 0;
-                z-index: 2;
+                z-index: 5; // Stack sidebar above annotations. See z-index values https://github.com/box/box-annotations/blob/master/src/common/BaseAnnotator.scss
```

Comment:
Should we make it higher? I'm not sure we ever want something at a higher z-index

Reply:
yeah that makes sense but also not sure what is a safe and intentional value to set it to. I went back n forth on some alternatives such as adding a `z-index` to the preview container so it creates a stacking context always contained under the sidebar

but ultimately I wasn't confident with how these changes might interact with floating components like tooltips, modals, menus, flyouts, etc. also as we move to blueprint components (coming soon™) how those might interact with the stacking contexts of preview and sidebar

at the moment this seems to be the safest thing and as we handle migrating UI elements to blueprint, we can revisit the staking contexts